tests/bulkwritecommand/bulkwritecommand-updateOne_error-009.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+MongoDB\Driver\BulkWriteCommand::updateOne() with multi:true prohibits sort option
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = create_test_manager();
+$bulk = new MongoDB\Driver\BulkWriteCommand;
+
+// @TODO: FAILED: Expected MongoDB\Driver\Exception\InvalidArgumentException, but no exception thrown!
+echo throws(function() use ($manager, $bulk) {
+    $bulk->updateOne(NS, ['x' => ['$gt' => 1]], ['$set' => ['y' => 11]], ['multi' => true, 'sort' => ['x' => 1]]);
+}, MongoDB\Driver\Exception\InvalidArgumentException::class), "\n";
```

Comment:
I don't know why we don't validate the `sort` option for `updateOne`.

Reply:
I'm not sure what you mean. The given sort order looks correct?

Reply:
Collection bulk write is throwing an exception with this options: https://github.com/mongodb/mongo-php-driver/blob/d3a079fb4d6f9173a0c81c0d0250d1304e0292a3/tests/bulk/bulkwrite-update_error-009.phpt#L12-L21

Reply:
Ah. I'm not sure if legacy bulk write even supports `sort`, I'll have to double-check.

Reply:
We can check that option `sort` is not used with `updateMany`.
PHPLIB has an [exception for that](https://github.com/mongodb/mongo-php-library/blob/a4ccccd9011bad978e6f0d52efd1a512ef64d35a/src/Operation/Update.php#L156C29-L158), we can add it to `ClientBulkWrite::updateMany` and `ClientBulkWrite::deleteMany` in https://github.com/mongodb/mongo-php-library/pull/1630

Reply:
Tracked in [PHPLIB-1670](https://jira.mongodb.org/browse/PHPLIB-1670)